viya_ark_library/k8s/sas_kubectl.py

```diff
@@ -119,14 +119,16 @@ def __init__(self, executable: Text = "kubectl", namespace: Text = None, global_
     def get_namespace(self) -> Text:
         return self.namespace
 
-    def do(self, command: Text, ignore_errors: bool = False) -> AnyStr:
+    def do(self, command: Text, ignore_errors: bool = False, ignore_warnings: bool = False) -> AnyStr:
```

Comment:
Could you explain what `ignore_warnings` is accomplishing here? I'm not sure I follow it's use.

It seems like it's just returning a value that could be retrieved by the CalledProcessError that was thrown. At most it seems like that boolean would only need to be used to hide what's printed to stdout, meaning:

`if not ignore_warnings:
      print(...)`

seems like better use, but I'm not sure why we want to hide warnings, it may give the user an idea of which logs to evaluate.

Reply:
The ignore_warnings is introduced for backward support.